### ISSUE TITLE
Growing the project

### ISSUE THREAD

**Issue by jan-janssen** [jan-janssen/latex-examples#4]
Hi @srmnitc 

When I saw you liked `jupyterlab-latex`, I started to take a deeper look at it. And created the conda package: 
https://github.com/conda-forge/jupyterlab_latex-feedstock

Today I finally had the time to integrate this package in this project, which lead to me identifying a few more issues in `jupyterlab-latex`:
https://github.com/jan-janssen/latex-examples/issues/3

And at least for one of these issues I was able to push the fix to the project as a pull request: 
https://github.com/jupyterlab/jupyterlab-latex/pull/133

Basically it is opensource in the way it should be. 

Still I am wondering if we should coordinate these activities a bit more - basically a joined collection of tools and software packages for materials scientists. 

Best, 

Jan

**Comment by srmnitc** [comment]
Hi @jan-janssen, a collection of tools and packages for material scientists is a great idea. In the times of working from home, I realise how using jupyter notebooks and the whole environment simplifies the workflow quite a bit. It would make sense to curate and collect a package. I would be happy to contribute.

**Comment by jan-janssen** [comment]
Ok, so maybe we start collecting here and then it most likely makes sense to start some separate Github group at some point: 

** DFT:**

- https://github.com/jan-janssen/quantum-espresso-example
- https://github.com/jan-janssen/gpaw-example
- https://github.com/jan-janssen/cp2k-example

** MD: **

- https://github.com/jan-janssen/lammps-example
- https://github.com/jan-janssen/openkim-example
- https://github.com/jan-janssen/kliff-example
- https://github.com/jan-janssen/atomicrex-example
- https://github.com/jan-janssen/ovito-example
- https://github.com/jan-janssen/ovito3-example

** Cluster expansion: **

- https://github.com/jan-janssen/clease-example
- https://github.com/jan-janssen/icet-example

** Thermo dynamic: **

- https://github.com/jan-janssen/phonopy-example
- https://github.com/jan-janssen/espei-example

** utils:**

- https://github.com/pyiron/pyiron
- https://github.com/jan-janssen/matminer-example 
- https://github.com/jan-janssen/mendeleev-example

** development: **

- https://github.com/jan-janssen/voila-example
- https://github.com/jan-janssen/code-server-example
- https://github.com/jan-janssen/mpi4py-example

** TeX: **

- https://github.com/jan-janssen/latex-examples


I was thinking about compiling a list like: 
https://github.com/tilde-lab/awesome-materials-informatics
Where for each software to be included in the list, they have to provide a mybinder repository.

**Comment by jan-janssen** [comment]
In addition I started to create gif based previews: 
![qe](https://user-images.githubusercontent.com/3854739/78343692-829e1400-759b-11ea-9a46-ad56e58f3bb7.gif)

Also for this I created a small app: 
https://github.com/jan-janssen/mov2gif

**Comment by srmnitc** [comment]
This is quite nice! Do we also include for example, jupyter lab extensions?

**Comment by jan-janssen** [comment]
Everything, I would suggest we start collecting first and do then the sorting later.

**Comment by jan-janssen** [comment]
I started with a collection here: https://matbinder.github.io and the source code is here https://github.com/matbinder
